_rules/meta-refresh-no-delay-no-exception-bisz58.md

```diff
@@ -0,0 +1,212 @@
+---
+id: bisz58
+name: '`meta` element has no refresh delay (no exception)'
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used for delayed redirecting or refreshing.
+accessibility_requirements:
+  wcag20:2.2.4: # Interruptions (AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:3.2.5: # Change on Request (AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G110: # Using an instant client-side redirect
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:H76: # Using meta refresh to create an instant client-side redirect
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'title-require'
+---
+
+## Applicability
+
+This rule applies to the first [valid](https://html.spec.whatwg.org/#attr-meta-http-equiv-refresh) `<meta http-equiv="refresh">` element with a `content` attribute in a document.
+
+## Expectation
+
+The `time` of the `content` attribute is 0.
+
+See the definition of the [Refresh state (`http-equiv="refresh"`)](https://html.spec.whatwg.org/#attr-meta-http-equiv-refresh) for a precise description on how to determine the `time`.
+
+## Assumptions
+
+- This rule assumes that no functionality was provided by the website for the user to adjust the timer.
+
+## Accessibility Support
+
+Not all major web browsers parse the value of the `content` attribute in the same way. This can cause redirects to happen in some browsers, but not in others. This can cause some pages to pass this rule, while still having a redirect in a minority of web browsers.
```

Comment:
I think this should be more explicit about what they do differently. Otherwise we're just leaving people guessing.

Reply:
I honestly don't know 😅 
Since you are an author of the original rule, do you remember what this AS note is referring to?

Reply:
I'll just leave this here: https://github.com/act-rules/act-rules.github.io/pull/1422/files. Not that I don't agree 🤷

Reply:
I don't recall it either. But is it really needed to say anything more than "some browsers redirect while others don't"? I don't think we need to explain why some browsers don't follow the spec.

Reply:
Mark has a couple examples: https://github.com/act-rules/act-rules.github.io/issues/1141#issue-558511928. From the look of it, If Firefox can't parse the value it default to a 0s delay. So:

```suggestion
Not all major web browsers parse the value of the `content` attribute in the same way. Some major browsers, if they are unable to parse the fault default to a 0s delay, whereas others will not redirect at all. This can cause some pages to be inapplicable for this rule, while still having a redirect in a minority of web browsers.
```